1.24.1

matteobruni opened the pull request:
fix: emitter play/pause were not working fine from container
feat: added autoPlay option to emitters
fix: confetti fixed oval rotation
feat: added drift (numeric or min/max object)
feat: added min/max alternative object to move.speed
feat: size, opacity and rotate particles options support now also min/max objects in value

matteobruni committed:
fix: emitter play/pause were not working fine from container
feat: added autoPlay option to emitters

ghost commented:
## DeepCode's analysis on [#e6db8e](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/217a573fc5364702fcc67a374e1ca6c7d29d281d/matteobruni/tsparticles/e6db8ec367110379ed1594b34b603d252f7fb478/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=1&) found:

- :information_source: **1** minor issue. :point_down:
- :heavy_check_mark: **1** issue was fixed.
## Top issues

<table>
  <thead>
    <tr>
      <th align="left">Description</th>
      <th align="left">Example fixes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td width="77%">'public' is implicit. Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/e6db8ec367110379ed1594b34b603d252f7fb478/core/main/src/Plugins/Emitters/EmitterInstance.ts#L31">EmitterInstance.ts:31</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/e6db8ec367110379ed1594b34b603d252f7fb478/core/main/src/Plugins/Emitters/EmitterInstance.ts#L32">EmitterInstance.ts:32</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/e6db8ec367110379ed1594b34b603d252f7fb478/core/main/src/Plugins/Emitters/EmitterInstance.ts#L33">EmitterInstance.ts:33</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/217a573fc5364702fcc67a374e1ca6c7d29d281d/matteobruni/tsparticles/e6db8ec367110379ed1594b34b603d252f7fb478/pr/_/%2Fcore%2Fmain%2Fsrc%2FPlugins%2FEmitters%2FEmitterInstance.ts/javascript%2Ftslint%2Fmember-access/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
  </tbody>
</table>


#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/217a573fc5364702fcc67a374e1ca6c7d29d281d/matteobruni/tsparticles/e6db8ec367110379ed1594b34b603d252f7fb478/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=1&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=matteobruni)

matteobruni committed:
fix: confetti fixed oval rotation

matteobruni committed:
build: updated install script

matteobruni committed:
feat: added drift (numeric or min/max object)
feat: added min/max alternative object to move.speed

matteobruni committed:
build: changed default confetti type to both

matteobruni committed:
feat: size, opacity and rotate particles options support now also min/max objects in value

matteobruni committed:
chore(release): published new version

 - ng-particles@2.7.0
 - inferno-particles@1.7.0
 - jquery-particles@1.24.0
 - preact-particles@1.24.0
 - react-tsparticles@1.24.0
 - svelte-particles@1.7.0
 - particles.vue@2.7.0
 - particles.vue3@1.7.0
 - tsparticles-editor@1.24.0
 - tsparticles@1.24.0
 - ng-particles-demo@1.9.0
 - editor-demo@1.9.0
 - inferno-particles-demo@1.7.0
 - jquery-particles-demo@1.9.0
 - tsparticles-demo@1.9.0
 - preact-particles-demo@1.9.0
 - react-tsparticles-demo@1.9.0
 - svelte-demo@1.7.0
 - particles.vue-demo@1.9.0
 - particles.vue3-demo@1.7.0
 - tsparticles-preset-60fps@1.9.0
 - tsparticles-preset-background-mask@1.9.0
 - tsparticles-preset-basic@1.9.0
 - tsparticles-preset-big-circles@1.9.0
 - tsparticles-preset-bouncing@1.9.0
 - tsparticles-preset-bubbles@1.9.0
 - tsparticles-preset-fire@1.9.0
 - tsparticles-preset-font-awesome@1.9.0
 - tsparticles-preset-fountain@1.9.0
 - tsparticles-preset-sea-anemone@1.9.0
 - tsparticles-preset-snow@1.9.0
 - tsparticles-preset-stars@1.9.0
 - tsparticles-shape-bubble@1.9.0
 - tsparticles-shape-confetti@1.9.0
 - tsparticles-shape-heart@1.9.0
 - tsparticles-shape-multiline-text@1.9.0
 - tsparticles-shape-rounded-rect@1.9.0
 - tsparticles-shape-spiral@1.9.0
 - cra-template-particles-typescript@1.7.0
 - cra-template-particles@1.7.0
 - tsparticles-website@1.24.0

matteobruni committed:
fix: fixed emitters after 1.24.0

matteobruni committed:
chore(release): published new version

 - ng-particles@2.7.1
 - inferno-particles@1.7.1
 - jquery-particles@1.24.1
 - preact-particles@1.24.1
 - react-tsparticles@1.24.1
 - svelte-particles@1.7.1
 - particles.vue@2.7.1
 - particles.vue3@1.7.1
 - tsparticles-editor@1.24.1
 - tsparticles@1.24.1
 - ng-particles-demo@1.9.1
 - editor-demo@1.9.1
 - inferno-particles-demo@1.7.1
 - jquery-particles-demo@1.9.1
 - tsparticles-demo@1.9.1
 - preact-particles-demo@1.9.1
 - react-tsparticles-demo@1.9.1
 - svelte-demo@1.7.1
 - particles.vue-demo@1.9.1
 - particles.vue3-demo@1.7.1
 - tsparticles-preset-60fps@1.9.1
 - tsparticles-preset-background-mask@1.9.1
 - tsparticles-preset-basic@1.9.1
 - tsparticles-preset-big-circles@1.9.1
 - tsparticles-preset-bouncing@1.9.1
 - tsparticles-preset-bubbles@1.9.1
 - tsparticles-preset-fire@1.9.1
 - tsparticles-preset-font-awesome@1.9.1
 - tsparticles-preset-fountain@1.9.1
 - tsparticles-preset-sea-anemone@1.9.1
 - tsparticles-preset-snow@1.9.1
 - tsparticles-preset-stars@1.9.1
 - tsparticles-shape-bubble@1.9.1
 - tsparticles-shape-confetti@1.9.1
 - tsparticles-shape-heart@1.9.1
 - tsparticles-shape-multiline-text@1.9.1
 - tsparticles-shape-rounded-rect@1.9.1
 - tsparticles-shape-spiral@1.9.1
 - cra-template-particles-typescript@1.7.1
 - cra-template-particles@1.7.1
